### ISSUE TITLE
1.9.4 Enviromental Tech basalt not breaking in 3x3

### ISSUE THREAD

**Issue by Ozzymud** [ColinGBos/Hammerz#15]
When breaking the center of a 3x3 wall of basalt, only the center block drops, the rest stay in place. Waila shows the effective tool as a pickaxe with no level restrictions. I checked with a wooden pick and it mines. 

Is there a way to tell the hammers to mine this in a 3x3? I looked at the config and don't see any place to add new blocks.  

forge:1.9.4-12.17.0.1965
1.9.4-Hammerz-1.9.4-0.0.1.1
environmentaltech-1.9.4-0.9.4c

Edit: seems to happen with other enviromental tech blocks as well, hardened stone does the same thing.

**Comment by ColinGBos** [comment]
Yeah I noticed this, what struck me was that when you mine with a higher mining level hammer it works perfectly fine which is why I think it might be on their end. They set the harvest level of the block to be something like iron but then still let the block drop when anything else breaks it. Hammers will look at the block type, harvest level and relative hardness when determining to break a block or not and it's all done automatically so if mods do everything correctly with their blocks it's worked well so far.
I'll check it out further though, thanks for the report

**Comment by Ozzymud** [comment]
As it is the only mod it happens with so far, make sense, I'll see about posting at their place.

**Comment by ValkyrieofNight** [comment]
I don't think this is an issue with Environmental Tech because the blocks are basically vanilla blocks. The mining level is default. Only material, block hardness and resistance has been set. 

Alabaster: hardness = 2.5f, resistance = 4.0f, Material = Rock
Basalt: hardness = 3.0f, resistance = 4.0f, Material = Rock
Hardened Stone: hardness = 4.0f, resistance = 4000000.0f, Material = Rock
Lonsdaleite: hardness = 3.5f, resistance = 9000000, Material = Iron

Could be the hardness?
